ledger/sandbox-on-x/reference.conf

```diff
@@ -351,6 +351,14 @@ ledger {
       #  not-before = 5
       #}
 
+      #jwt-audience {
+      #  # Enable audience-based JWT verification.
+      #  enabled = true
```

Comment:
why do we need two fields here? Can we not make this an optional and if it’s set its enabled otherwise it's disabled?

Reply:
without having `enabled` separately and stacking multiple config files together, it would be impossible to "undo" the audience change and disable it on the layer above, that is why we are using this pattern of "enabled" config key

Reply:
Do we do that for other config values as well? I haven't seen that being used in the Canton config.

Reply:
we do that for the SoX configuration, but not for the canton. Probably this issue has not been yet discovered there ;)